netmap/node_info.go

```diff
@@ -460,6 +460,29 @@ func (x NodeInfo) IterateAttributes(f func(key, value string)) {
 	}
 }
 
+// GetAttributes returns all the node attributes.
+//
+// See also Attribute, IterateAttributes.
+func (x NodeInfo) GetAttributes() []netmap.Attribute {
```

Comment:
i dont think it is what was discussed: you still require using `netmap` from api-go by a caller (even if you do it implicitly). i guess you need to return some SDK struct or some builtin type, no api-go ever. however, @cthulhu-rider, dropped the former `Attribute` struct, so we need to do smth here

Reply:
we can use native `[][2]string` for this, or we can share `KV struct { K, V string}` b/w container, object, node etc. I like 1st more

Reply:
`map[string]string`? Do we care about order?

Reply:
order matters in Set for https://pkg.go.dev/github.com/nspcc-dev/neofs-sdk-go/netmap#NodeInfo.Marshal. Get may return a map

Reply:
Did `[][2]string` for Get and Set.